packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+meta:
+  - name: description
+    content: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+layout: Guides
+---
+
+This guide — part of the [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview/) series — teaches you how to create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+
+## Getting connected
+
+After you have a SCIM implementation that passes all of the Runscope tests, as discussed in the [previous part of the series](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api), you need to create your SCIM integration directly within Okta.
+
+Begin by signing up for an [Okta developer account](https://developer.okta.com/signup/).
+
+1. After you request the developer account and have received the initial email, click the link to go to your Admin Console.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Browse App Catalog**.
+1. Search for "SCIM 2.0" or "SCIM 1.1" (your choice depends on which version your SCIM server supports). You'll see three different SCIM template applications, one for each of the three authentication methods that you can use to connect to your SCIM implementation (Basic Auth, Header Auth, or OAuth Bearer Token).
```

Comment:
How about (see the next comment for a suggestion for the follow on text):
1. Search for either "SCIM 2.0" or "SCIM 1.1" depending on the version of SCIM supported by your server.

Reply:
done

Reply:
updated.